src/firefly/js/ui/tap/TableSearchMethods.jsx

```diff
@@ -727,36 +803,65 @@ function tapSearchMethodReducer(columnsModel) {
 
                     rFields[timeKey] = clone(inFields[timeKey], {value, message, valid, showHelp, feedback,
                         timeMode: ISO});
+                    validateTemporalConstraints(inFields, rFields);
                 }
             };
 
+            const onChangeSearchCheckBox = (key) => {
+                if (key === SpatialCheck) {
+                    return validateSpatialConstraints(inFields, rFields, value !== Spatial );
+                } else if (key === TemporalCheck) {
+                    return validateTemporalConstraints(inFields, rFields, value !== Temporal );
+                }
+            };
+
+            const onResetColumnsTable = () => {
+                const cols = getAvailableColumns(columnsModel);
+                set(rFields, [CrtColumnsModel, 'value'], columnsModel.tbl_id );
+                const centerColObj = formCenterColumns(columnsModel);
+                Object.assign(rFields[CenterLonColumns],
+                                       {validator: getColValidator(cols, false), value: centerColObj.lon, valid: true});
+                Object.assign(rFields[CenterLatColumns],
+                                       {validator: getColValidator(cols, false), value: centerColObj.lat, valid: true});
+
+                Object.assign(rFields[TemporalColumns],
+                                        {validator: getColValidator(cols, false), value: '', valid: true});
```

Comment:
The third argument to getColValidator should be false, if we don't want columns expressions. (The default is true.)

Reply:
updated. thanks.